packages/hardhat-ganache/test/index.ts

```diff
@@ -51,16 +51,15 @@ describe.skip("Ganache plugin with empty configs", function () {
   });
 });
 
-// Note: Skipping as this plugin hasn't been updated to the latest Ganache version,
-//   so it doesn't work well with Node 18
-describe.skip("Ganache plugin with custom configs", function () {
+// Note: Each of the skipped test is working when ran with .only
```

Comment:
@fvictorio Please help me out on this. I don't understand why these tests work when ran alone but not with the other tests.

The first two skipped tests say `HardhatError: HH5: HardhatContext is not created.` and the last one `AssertionError: expected 1 to equal +0`

Reply:
@fvictorio they do work when ran alone